data/json/items/armor.json

```diff
@@ -4351,7 +4351,7 @@
         "phase" : "solid",
         "encumbrance" : 15,
         "bashing" : 6,
-        "flags" : ["VARSIZE", "STURDY", "OUTER"],
+        "flags" : ["VARSIZE", "POCKETS", "STURDY", "OUTER"],
```

Comment:
The pockets are most likely unsuited to holding hands in them - they're made for armor plates and thus unlikely to provide the warmth the vest itself provides and also located in awkward locations.
Same for all the variants with armor inserted.